content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -162,13 +135,23 @@ The Istio control plane must be installed with virtual machine integration enabl
     the scope of this guide.
     {{< /idea >}}
 
+1. Create `sidecar.env` to import required environment variable
```

Comment:
some of this is duplicate with line 147.

Reply:
this is not duplicate as the PROV_CERT and OUTPUT_CERTS must be provided to enable the cert rotation feature